### PR TITLE
feat(tabset): tabs horizontal alignment

### PR DESCRIPTION
**Pull request by ktriek**
# Tabs Horizontal alignment
Add the horizontal alignment support to the tabs

- Start (Default)
- Center
- End

http://v4-alpha.getbootstrap.com/components/navs/#horizontal-alignment

### CODE REVIEW

File: src/tabset/tabset.ts [hunk 1]
```diff
@@ -110,11 +110,16 @@ export class NgbTabset implements AfterContentChecked {
   @Input() type: 'tabs' | 'pills';
 
   /**
+   * The horizontal alignment of the nav with flexbox utilities. Can be one of 'start', 'center' or 'end'
+   */
+  @Input() alignment: 'start' | 'center' | 'end';
```
Comment (pkozlowski-opensource) [inline src/tabset/tabset.ts]: How about calling it `justify`?

**Comment by pkozlowski-opensource** [comment]
Thnx for the PR. Could you please fix the linting issue? It fails the build currently: https://travis-ci.org/ng-bootstrap/ng-bootstrap/builds/191182918

File: src/tabset/tabset.ts (hunk 1 above)
Comment (ktriek) [inline src/tabset/tabset.ts]: I had to pick between justify, align and alignement , @pkozlowski-opensource  should we change the name to justify?

Comment (pkozlowski-opensource) [inline src/tabset/tabset.ts]: My preference would go to `justify` so yes, let's change it.

File: src/tabset/tabset-config.ts [hunk 2]
```diff
@@ -8,4 +8,5 @@ import {Injectable} from '@angular/core';
 @Injectable()
 export class NgbTabsetConfig {
   type: 'tabs' | 'pills' = 'tabs';
+  justify: 'start' | 'center' | 'end' = 'start';
```
Comment (maxokorokov) [inline src/tabset/tabset-config.ts]: Please order fields alphabetically

File: src/tabset/tabset.ts [hunk 3]
```diff
@@ -110,11 +110,19 @@ export class NgbTabset implements AfterContentChecked {
   @Input() type: 'tabs' | 'pills';
 
   /**
+   * The horizontal alignment of the nav with flexbox utilities. Can be one of 'start', 'center' or 'end'
+   */
+  @Input() justify: 'start' | 'center' | 'end';
```
Comment (maxokorokov) [inline src/tabset/tabset.ts]: Please order inputs alphabetically

**Comment by pkozlowski-opensource** [comment]
Merged, thnx

**Pull request closed without merging**